src/backend/database_migrations/versions/20220914_205721_make_sample_pathogen_required.py

```diff
@@ -0,0 +1,38 @@
+"""make sample pathogen required
+
+Create Date: 2022-09-14 20:57:27.921683
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20220914_205721"
+down_revision = "20220909_232743"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # backpopulate pathogen fields to sars-cov-2
+    op.execute(
+        """
+        UPDATE aspen.samples
+        SET pathogen_id=(
+            SELECT id from aspen.pathogens WHERE name='SARS-CoV-2'
+        )
+        WHERE pathogen_id IS NULL
+        """
+    )
```

Comment:
do we also need to do this for phylo_runs and phylo_trees?

Reply:
Like this? 

https://github.com/chanzuckerberg/czgenepi/blob/trunk/src/backend/database_migrations/versions/20220907_225235_add_pathogens_to_all_trees.py

Reply:
Very minor nit: feels like we should use `WHERE slug='SC2'` if that's how the other migration did it?